students/meeryu/users/views.py

```diff
@@ -35,3 +35,16 @@ def post(self,request):
         except KeyError:
             return JsonResponse({"message" : "KEY_ERROR"}, status = 400) 
             
+class SignInView(View):
+    def post(self,request):
+            try: 
+                data     = json.loads(request.body)
+                email    = data['email']
+                password = data['password']
+                
+                if not User.objects.filter(email = email, password = password):
+                    return JsonResponse({"message" : "INVALID_USER"}, status = 401)
+                return JsonResponse({"message" : "SUCCESS"}, status=201)
+
+            except KeyError:
+                return JsonResponse({"message":"KEYERROR"}, status = 400)
```

Comment:
다른 에러 메세지와 마찬가지로 통일된 양식으로 작성해주시면 더 좋을 것 같아요 미류님!
```suggestion
                return JsonResponse({"message":"KEY_ERROR"}, status = 400)
```